### PR TITLE
Feat/empty

### PR DESCRIPTION
**Pull request by kooriookami**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/empty/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@element-plus/empty",
+  "version": "0.0.0",
+  "main": "dist/index.js",
+  "license": "MIT",
+  "peerDependencies": {
+    "vue": "^3.0.3"
```
Comment (JeremyWuuuuu) [inline packages/empty/package.json:7]: Change to `^3.0.5`

File: packages/empty/src/img-empty.vue [hunk 2]
```diff
@@ -0,0 +1,128 @@
+<template>
+  <svg
+    width="79px"
+    height="86px"
```
Comment (JeremyWuuuuu) [inline packages/empty/src/img-empty.vue:4]: I think these 2 attributes can be removed. Could you remove it and see if it works?

File: packages/empty/src/img-empty.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+      />
+    </defs>
+    <g
+      id="Illustrations"
+      stroke="none"
+      stroke-width="1"
+      fill="none"
+      fill-rule="evenodd"
+    >
+      <g id="B-类空状态通用插图" transform="translate(-1268.000000, -535.000000)">
```
Comment (JeremyWuuuuu) [inline packages/empty/src/img-empty.vue:47]: Replace this to English characters.

File: packages/empty/src/index.vue [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="el-empty">
+    <div class="el-empty__image" :style="imageStyle">
+      <img v-if="image" :src="image" ondragstart="return false">
+      <slot v-else-if="$slots.image" name="image"></slot>
+      <img-empty v-else />
```
Comment (JeremyWuuuuu) [inline packages/empty/src/index.vue:6]: Change it this:

```suggestion
<slot name="image">
  <img-empty />
</slot>
```

**Review by jw-foss — CHANGES_REQUESTED** [review]
Commented above which you need to address.

File: packages/empty/src/index.vue [hunk 5]
```diff
@@ -2,8 +2,9 @@
   <div class="el-empty">
     <div class="el-empty__image" :style="imageStyle">
       <img v-if="image" :src="image" ondragstart="return false">
```
Comment (JeremyWuuuuu) [inline packages/empty/src/index.vue:4]: `@dragstart`?

**Review by jw-foss — APPROVED** [review]

**Comment by jikoy** [comment]
Where is the demo ?

**Comment by zazzaz** [comment]
@jikoy https://element-plus.org/#/en-US/component/empty

**Pull request merged**